web/packages/teleport/src/Notifications/Notification.tsx

```diff
@@ -165,11 +169,12 @@ function formatDate(date: Date) {
 }
 
 const Container = styled.div`
+  box-sizing: border-box;
   display: flex;
   align-items: center;
   justify-content: flex-start;
   gap: ${props => props.theme.space[3]}px;
-  max-width: 400px;
+  width: 100%;
```

Comment:
could this container grow super large somehow now?

Reply:
Yes, but this `Notification` component will only ever be used in a list, and so it'll be the responsibility of the parent component (the list component) to determine how wide it wants to be and the notifications in it will adapt accordingly.